components/brave_ads/core/internal/user_engagement/ad_events/new_tab_page_ads/new_tab_page_ad_event_handler_delegate.h

```diff
@@ -25,13 +25,23 @@ class NewTabPageAdEventHandlerDelegate {
   // Invoked when the new tab page `ad` is clicked.
   virtual void OnDidFireNewTabPageAdClickedEvent(const NewTabPageAdInfo& ad) {}
 
+  // Invoked when the new tab page `ad` is interacted with.
```

Comment:
what counts as an interaction?

Reply:
The user initiated a mouse-down event within a hit box of a rich media interactive element.

Reply:
not blocking but just making sure, does every interaction also fire the click event?

Reply:
It does not. Clicks are only fired when clicking on an element that opens the target url of the ad.